src/frontend/src/catalog/table_catalog.rs

```diff
@@ -95,7 +95,7 @@ pub struct TableCatalog {
     /// Owner of the table.
     pub owner: UserId,
 
-    /// Properties of the table. For example, `appendonly` or `retention_seconds`.
+    /// Properties of the table. For example, `appendonly`, `retention_seconds` and `timezone`.
```

Comment:
It's becoming a little freaky...

`appendonly`, `retention_seconds` are table's properties for sure, but `timezone` is actually a property for materialize view's plan (the streaming job), instead of the materialize view's results (the table)

`FragmentManager` seems to be a better place to store such information. What do you think? @st1page @yezizp2012

Reply:
Definitely doable. Agreed that timezone is not a property of table.

Reply:
Note that `TableFragments` is meant to represent not only `MaterializedView` but also `Table (source)`, `Sink` and `Index`.

Reply:
Yes, because they are also streaming jobs 😄

Reply:
Maybe we should add an MViewCatalog structure :crying_cat_face: 
actually, I think the `appendonly` have a slightly different meaning on table and MView(or streaming)

Reply:
+1 to place `timezone` in `TableFragments`. Actually it's better to rename `TableFragments` to `StreamingFragments`. NTFS 🥵